HW7/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+exports.default = {
+    mode: process.env.NODE_ENV || 'development',
+    entry: './src/index.js',
+    output: {
+        path: path.resolve(__dirname, './dist'),
+        clean: true,
+        filename: 'bundle.js'
+    },
+    module: {
+        rules: [
+            // {
+            //     test: /\.ts$/,
+            //     use: 'ts-loader',
+            //     //include: [path.resolve(__dirname,'src')]
+            // },
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader'],
+            },
+            {
+                test: /\.(png|svg)$/,
+                use: 'file-loader'
+            },
+        ],
+    },
+    // resolve: {
+    //     extensions: ['.ts'],
+    // },
+    plugins: [
+        new HtmlWebpackPlugin({ template: 'dist/index.html'}),
```

Comment:
dist is the folder for files generated by the webpack. There shouldn't be any of the source files. You should put the template to the src/ folder ad remove dist/ from the github